xfce4-settings-qubes.spec.in

```diff
@@ -1,73 +1,66 @@
 Name:		xfce4-settings-qubes
-Version:	4.0.3
-Release:	2%{?dist}
+Version:	@VERSION@
+Release:	@REL@%{?dist}
 Summary:	Default Xfce4 panel settings for Qubes
 
 Group:		User Interface/Desktops
 License:	GPLv2+
 URL:		http://www.qubes-os.org/
-Source0:	xfce4-panel-qubes-default.xml
-Source2:	xsettings.xml
-Source3:	xfwm4.xml
-Source4:	xfce4-desktop.xml
-Source5:	xfce4-session.xml
-Source6:	xfce4-power-manager.xml
-Source7:	xfce4-keyboard-shortcuts.xml
-Source8:	xfce4-xss-lock.desktop
-Source9:	qubes-update-xfce-config.desktop
-Source10:	update-xfce-config
+Source0:	%{name}-%{version}.tar.gz
 
 Requires:	qubes-artwork
 Requires:	xfce4-panel
 Requires:	xss-lock
 Requires(post):	xfce4-panel
 
+# We ship garcon menu file into this package
+# the fork is not needed anymore
+Conflicts:  garcon = 1000:0.6.4
+Requires: garcon >= 0.6.2
```

Comment:
This doesn't work for me. On update (`qubes-dom0-update --clean`) I get:
```
 Problem: package xfce4-settings-qubes-4.0.3-2.1.fc32.x86_64 requires garcon >= 0.6.2, but none of the providers can be installed
  - cannot install the best update candidate for package garcon-1000:0.6.4-4.fc32.x86_64
  - package xfce4-settings-qubes-4.0.3-2.1.fc32.x86_64 conflicts with garcon = 1000:0.6.4 provided by garcon-1000:0.6.4-4.fc32.x86_64
  - problem with installed package xfce4-settings-qubes-4.0.3-2.fc32.x86_64
  - cannot install the best update candidate for package xfce4-settings-qubes-4.0.3-2.fc32.x86_64
```

I've tried also with `--allowerasing`, `--best`, `--nobest`, `--action=distro-sync` and nothing helped. BTW `--best --allowerasing` is [hardcoded on the VM side](https://github.com/QubesOS/qubes-core-agent-linux/blob/master/package-managers/qubes-download-dom0-updates.sh#L62), which makes `--nobest` refused.